src/transformers/file_utils.py

```diff
@@ -351,6 +351,24 @@ def is_torch_bf16_available():
         return False
 
 
+def is_torch_tf32_available():
+    if is_torch_available():
+        import torch
```

Comment:
Tis is just a styling nit, but since we do a bunch of if not, return False afterward, could we remove that big test and begin with
```py
    if not is_torch_available():
        return False

    import torch
```
and then unindent all the other tests and remove the else part?

Reply:
great suggestion, Sylvain. done.